CoC.md

```diff
@@ -114,6 +114,77 @@ following behaviors:
+are no exceptions for those in positions of power such as maintainers,
+sponsors, funders, or other individuals withe decision making authority.
+Further, people in positions of power can wield it to exacerbate the effects of
+harassment and to diminish the repercussions, those who are informal or formal
+leaders are held to a higher standard.
+
+Anyone asked by another community member to stop unacceptable behavior is
+expected to comply immediately. However, you should not step in on someone
+else's behalf without their consent.
+
+### How to get help
+
+The Community Council is made up established members of the community who
+assist with resolving conflicts within the community.
+
+Currently the Community Council consists of:
+
+- Jen Helsby (`@redshiftzero`) - Lead Engineer - [jen@freedom.press](mailto:jen@freedom.press)
+- Loic Dachary (`@dachary`) - Maintainer - [loic@freedom.press](mailto:loic@freedom.press)
+
+You can contact the whole Community Coucil or members individually.
+
+You should contact the Community Council if you have questions or concerns about
+the Code of Conduct (including improvements) or if you feel that you have
+witnessed a Code of Conduct violation. In the even of a violation either
+directed at yourself or someone else, please contact the Community Council as
+soon as possible through whatever analog or digital medium you are most
+comfortable with.
+
+Should report be about any member of the Community Council or if there is a
+conflict of interest, that member will recuse themselves from the conflict
+resolution proceedings. They will not be involved with the discussion,
+documentation, communications, or decisions made by the rest of the Community
+Council in regards to the incident.
+
+## What the person reporting a violation can expect
+
+The Community Council will respond to reports as quickly as possible. When
+responding to a report, the Community Council will prioritize the safety of the
+person(s) who have been harmed or are at risk of harm and the reporter(s). No
+actions will be taken without the consent of the person who has been harmed or
+is at risk of harm except in cases where danger or harm are imminent.
+
+All reports to the Community Council, no matter how minor or severe, will be
+taken seriously and looked into.
+
+## How the Community Council responds to incidents
+
+The Community Council does not have a fixed set responses to some enumerated set
+of incidents that may occur. The Community Council operates on a case-by-case
+basis taking into account past behavior; the relationship between the person(s)
+who were harmed and the person(s) causing the harm; the responses of the
+person(s) who caused harm; and the perceived threat of future harm.
+
+Actions the Community Council may take to mitigate harm includes, but is not
+limited to:
+
+- A simple warning
+- A temporary ban from email lists, chat channels, repositories, or other online
+  communication mediums
+- Informal mediation
```

Comment:
I would move this to or near the top of the list, as the list generally seems to imply escalating consequences.